tests/test_util.py

```diff
@@ -0,0 +1,58 @@
+import unittest
+
+from pacai.util import util
+
+"""
+This is a test class to assess the functionality of the data structures 
+defined in util.py. The data structures to be tested are the Queue, Stack,
+and the Priority Queue
+"""
+class UtilTest(unittest.TestCase):
+
+    def test_queue(self):
+
+        queue = util.Queue()
+        self.assertTrue(queue.isEmpty())
+
+        random_val_list = [hash(x) for x in range(1, 10)]
```

Comment:
Why are you doing hash? This is an int, so hash will just return the int value.

Reply:
I just wanted to fill the list with random values, no significant reason as to why I used hash().

Reply:
hash() on an int just returns the int's value.

Reply:
Also, be VERY careful using random values in tests. Avoid it whenever you can.
For example, you also use hash() in the priority queue test below. If hash() did return an arbitrary value (say an address), then that test would randomly fail (we call it a flaky test).

Reply:
Understood